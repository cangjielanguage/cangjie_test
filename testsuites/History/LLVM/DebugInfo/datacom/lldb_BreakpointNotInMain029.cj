// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_BreakpointNotInMain029_expected.txt ../basecase/042_debuginfun_expressions.cj
// EXEC: cjc -g 042_debuginfun_expressions.cj
// EXEC: python3 DatacomTest.py lldb_BreakpointNotInMain029_expected.txt



