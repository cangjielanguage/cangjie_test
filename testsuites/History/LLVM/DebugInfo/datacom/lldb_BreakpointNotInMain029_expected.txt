lldb,lldb 042_debuginfun_expressions,042_debuginfun_expressions
b,b 25,042_debuginfun_expressions.cj:25
b,b 26,042_debuginfun_expressions.cj:26
r,r,-> 25
c,c,-> 26
n,n,-> 29
cj_plocals,cj_plocals,weiq = -120